src/main/java/ca/uhn/fhir/jpa/starter/dotbase/accesslog/AccessLogProvider.java

```diff
@@ -75,4 +77,31 @@ private StringType logToStringType(AccessLog log) {
       log.timestamp;
     return new StringType(concat);
   }
+
+  @Operation(name = "$recent-patients", idempotent = true)
+  public Bundle getRecentPatients(
+    HttpServletRequest theRequest,
+    RequestDetails requestDetails
+  )
+    throws Exception {
+    String username = requestDetails.getAttribute("_username").toString();
+    List<PatientAccess> logs = ACCESS_LOG_MODEL.getRecentPatients(username);
+    return recentPatientsResponseBundle(logs);
+  }
+
+  private Bundle recentPatientsResponseBundle(List<PatientAccess> logs) {
+    Bundle res = new Bundle();
```

Comment:
```suggestion
    Bundle res = new Bundle();
    res.setType(BundleType.COLLECTION);
```
Bundle normally should have a mandatory attribute type. For custom operations FHIR spec suggests to use "Collection".